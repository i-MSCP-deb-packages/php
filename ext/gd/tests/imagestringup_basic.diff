001+ 6542f1c17613497bc9a4372cf5f38037
001- 7c28016adcf620b772af2a8655b87bd2