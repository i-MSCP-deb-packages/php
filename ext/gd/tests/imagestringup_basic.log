
---- EXPECTED OUTPUT
7c28016adcf620b772af2a8655b87bd2
---- ACTUAL OUTPUT
6542f1c17613497bc9a4372cf5f38037
---- FAILED
