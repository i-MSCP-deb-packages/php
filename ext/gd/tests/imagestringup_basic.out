6542f1c17613497bc9a4372cf5f38037